client/signup/config/flows.js

```diff
@@ -88,6 +89,13 @@ function filterDestination( destination, dependencies ) {
 }
 
 function getDefaultFlowName() {
+	if (
+		config.isEnabled( 'signup/onboarding-flow' ) &&
+		abtest( 'designFirstSignup' ) === 'designFirst'
```

Comment:
Is this the right place to do the test split? My concern is getting skewed assignment. Do we have any direct links to `/start/onboarding`?